nautobot_ssot/views.py

```diff
@@ -24,7 +24,7 @@
 class DashboardView(ObjectListView):
     """Dashboard / overview of SSoT."""
 
-    queryset = Sync.objects.all()
+    queryset = Sync.objects.defer("diff").all()
```

Comment:
Same comment as above should apply here if it applies there.

Reply:
same as above, `.values()` provides tuples, not objects.